TreeMaker/python/makeJetVars.py

```diff
@@ -328,6 +330,8 @@ def makeJetVarsAK8(self, process, JetTag, suff, storeProperties):
         JetPropertiesAK8.bDiscriminatorCSV = cms.vstring('pfBoostedDoubleSecondaryVertexAK8BJetTags')
         JetPropertiesAK8.subjets = cms.vstring('SoftDropPuppi')
         JetPropertiesAK8.SJbDiscriminatorCSV = cms.vstring('SoftDropPuppi','pfCombinedInclusiveSecondaryVertexV2BJetTags')
+        JetPropertiesAK8.tDiscriminatorDeep = cms.vstring('tDiscriminatorDeep')

```

Comment:
The vstring value should be `'deepAK8:tDiscriminatorDeep'` (and similar for the next one)